jobs/jobscheduler.py

```diff
@@ -349,7 +367,7 @@ def initJobs(jobs, resources, otype):
                 if (opts.maxmemory > 0)
                 else int(MachineMemoryGB * 1024 * 1024 * 10.24 * opts.memory)
             ),
-            "gpu": list(range(8)),
+            "gpu": CUDAGPUList + ROCMGPUList,
```

Comment:
@iarspider , though for now this is going to work but what will happen if we have a host with both nvidia and amd gpus? On such nodes though cmssw is only going to use either rocm or cuda but job scheduler here will keep on submitting jobs as gpu list here will have combined list of gpus. 

I would suggest to to replace `gpu` with `cuda` and `rocm` and if command has `HIP_VISIBLE_DEVICES=[0-9]+` then use `rocm` and if command has `CUDA_VISIBLE_DEVICES=[0-9]*` then use cuda resources

Reply:
as discussed our mattermost, the current logic to assigning any available gpu matches what runTheMatrix does. So no need to break `gpu` in to `cuda/rocm`